packages/block-library/src/navigation/edit/placeholder/index.js

```diff
@@ -31,6 +31,7 @@ const ExistingMenusDropdown = ( {
 	onFinish,
 	menus,
 	onCreateFromMenu,
+	canUserCreateNavigation = false,
```

Comment:
Same as above, this component doesn't need to know about permissions, I would rename this to `withCreateButton` as it pinpoints the purpose of this argument, whereas `canUserCreateNavigation` can potentially have many different effects.

Reply:
Fixed 👍